tests/go_test.go

```diff
@@ -359,7 +365,7 @@ func CheckMutateBuffer(org []byte, offset flatbuffers.UOffsetT, fail func(string
 		testcase{"Pos.Y'", func() bool { return monster.Pos(nil).Y() == float32(20.0) }},
 		testcase{"Pos.Z'", func() bool { return monster.Pos(nil).Z() == float32(30.0) }},
 		testcase{"Pos.Test1'", func() bool { return monster.Pos(nil).Test1() == float64(30.0) }},
-		testcase{"Pos.Test2'", func() bool { return monster.Pos(nil).Test2() == int8(20) }},
+		testcase{"Pos.Test2'", func() bool { return monster.Pos(nil).Test2() == example.Color(20) }},
```

Comment:
Please also check that `Pos(nil).Test2()` is equal to the strongly-type enum value `example.ColorGreen`. (And add the same logic for the other test, too.)

Reply:
Does this look better now? There's at least one line now that tests `Test2()` against each of a defined `Color` enum value and an int cast to a `Color`.

Reply:
Thanks, yeah! Please see my new review comment, too. As far as testing goes, I think I'm most concerned that we test a) the `ColorX` enum values, and b) the casting of _valid_ enum numbers to enum values. The casting of invalid enum numbers to enum values is useful, too, I guess, but not as critical.